site/content/docs/5.2/migration.md

```diff
@@ -58,6 +58,9 @@ Learn more by reading the new [color modes documentation]({{< docsref "/customiz
 
 - <span class="badge text-warning-emphasis bg-warning">Deprecated</span> The `.list-group-variant()` mixin is now deprecated. We now [use the Sass loop]({{< docsref "/components/list-group#sass-loop" >}}) directly to modify the component's default CSS variables for each variant.
 
+- **Carousel**
+  - <span class="badge bg-warning">Warning</span> The pause/play button has been slightly changed for accessibility purpose; it is now placed first and is the first focusable item.
```

Comment:
This is actually breaking change IMO since people will loose their button if they don't change the place.